### PR TITLE
Rename Action plugins

### PR DESCRIPTION
**Pull request by sreevatsanraman**
Renamed the following:
- DatabaseAction -> Database
- SSHAction -> SSH
- HDFSFIleMoveAction -> HDFSMove
- HDFSFileDeleteAction -> HDFSDelete

Build: http://builds.cask.co/browse/HYP-BAD81

Screenshots:

screen shot 2016-08-20 at 1 45 40 pm
screen shot 2016-08-20 at 1 45 46 pm
screen shot 2016-08-20 at 1 45 54 pm
screen shot 2016-08-20 at 1 46 04 pm
screen shot 2016-08-20 at 1 46 13 pm

### CODE REVIEW

File: core-plugins/docs/HDFSMove-action.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ Example
 This example moves a file from `/source/path` to `/dest/path`:
 
     {
-        "name": "HDFSAction",
```
Comment (anwar6953) [inline core-plugins/docs/HDFSMove-action.md:36]: rename it in the description also?
https://github.com/caskdata/hydrator-plugins/blob/78eac8424dcc51d6f6056ab2598594e930dec18a/core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/HDFSMoveAction.java#L149-L149

Comment (anwar6953) [inline core-plugins/docs/HDFSMove-action.md:36]: And in the `Properties` section of: HDFSMove-action.md

File: core-plugins/docs/SSH-action.md [hunk 2]
```diff
@@ -40,9 +40,9 @@ This example runs a script on demo@example.com:
     {
         "name": "SSHAction",
           "plugin": {
-            "name": "SSHAction",
```
Comment (anwar6953) [inline core-plugins/docs/SSH-action.md:43]: should we rename the `name` field of the example config.json?
https://github.com/caskdata/hydrator-plugins/blob/bd71d29fbbc6699c21f9aa9716c7d197fa6201d9/core-plugins/docs/SSH-action.md#L41-L41

File: core-plugins/docs/HDFSMove-action.md (hunk 1 above)
Comment (sreevatsanraman) [inline core-plugins/docs/HDFSMove-action.md:36]: fixed.

File: core-plugins/docs/SSH-action.md (hunk 2 above)
Comment (sreevatsanraman) [inline core-plugins/docs/SSH-action.md:43]: fixed

File: core-plugins/docs/SSH-action.md [hunk 3]
```diff
@@ -40,9 +40,9 @@ This example runs a script on demo@example.com:
...
+            "name": "SSH",
             "type": "action",
-            "label": "SSHAction",
+            "label": "SSH",
```
Comment (albertshau) [inline core-plugins/docs/SSH-action.md:45]: we should remove label. its not used by the backend at all and is not in any of the other examples.

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/HDFSFileDeleteAction.java [hunk 4]
```diff
@@ -42,7 +42,7 @@
  * Optionals include fileRegex
  */
 @Plugin(type = Action.PLUGIN_TYPE)
-@Name("HDFSFileDeleteAction")
+@Name("HDFSFileDelete")
```
Comment (albertshau) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/HDFSFileDeleteAction.java]: HDFSDelete

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/HDFSFileMoveAction.java [hunk 5]
```diff
@@ -42,7 +42,7 @@
  * Optionals include fileRegex
  */
 @Plugin(type = Action.PLUGIN_TYPE)
-@Name("HDFSFileMoveAction")
+@Name("HDFSFileMove")
```
Comment (albertshau) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/HDFSFileMoveAction.java]: HDFSMove

**Comment by albertshau** [comment]
lgtm

**Comment by sreevatsanraman** [comment]
Will merge once the build passes.

**Comment by sreevatsanraman** [comment]
Build passed merging.

**Pull request merged**